trigger/wsclient/metadata.go

```diff
@@ -3,6 +3,10 @@ package wsclient
 // Settings for the websocket client trigger
 type Settings struct {
 	URL string `md:"url,required"`
+	AllowInsecure bool   `md:"allowInsecure"`
+	CaCert        string `md:"caCert"`
+	QueryParams map[string]interface{} `md:"queryParams"`
```

Comment:
Change to map[string]string for query and header

Reply:
done